pkg/bot/socketslack.go

```diff
@@ -360,6 +371,13 @@ func (b *SocketSlack) send(event socketSlackMessage, req string, resp interactiv
 	//if the message is from thread then add an option to return the response to the thread
 	if event.ThreadTimeStamp != "" {
 		options = append(options, slack.MsgOptionTS(event.ThreadTimeStamp))
+	} else if file != nil {
+		for _, share := range file.Shares.Public {
+			if share[0].Ts != "" {
```

Comment:
> Please add one more check to ensure we will not panic here 🙂
> 
> adding multi `slack.MsgOptionTS(share[0].Ts)` option doesn't make sens. Only one be used (the last one), so to avoid misunderstanding, let's add only one option 👍
> 
> in the comment above, I suggested an alternative approach.

Actually, once you send a message to slack, by default it always adds share section as the original message. However, maybe for some other cases it can be nil, let me add to be on the safe side :)